wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,64 @@
+# WNT API Examples
+
+This folder contains the WNT API Examples.
```

Comment:
Extend WNT to Wirepas Network Tool and clarify which API we are using (client websocket API).

Provide a brief explanation of the API.

Reply:
Done